content/en/event-streaming.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Event Streaming
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Event streaming is an approach where [microservices](/microservices/) continuously communicate to the system what they are doing. 
+Picture a service broadcasting everything it does to all other services. 
+Each activity taken by a service is referred to as an event, hence event streaming. 
+For example, the Netflix service in charge of registering user movie choices might "broadcast" an event communicating something like: "User X selected movie Y."
+The message is now "out there" for the Netflix service in charge of pulling the movie from the database to pick it up.
+
+## Problem it addresses
+
+The traditional communication approach for services is "ask, don't tell," coupling services to one another. 
+In our example, the Netflix service that pulls the movie from the database would have to ask the service that registers user choices: 
+"What's your current state? Any movies I need to get?" 
+This approach assumes that the first service is always available to respond.
```

Comment:
This is IMHO not the problem statement, that is solved with Event Streaming. If one service relies on another service, which is not available to respond, event streaming won't help. High availability through horizontal scaling and auto healing will.

Reply:
Oh, so without event streaming HA, horizontal scaling , and self-healing isn't feasible?

Reply:
HA, horizontal scaling and self healing have nothing to do with event streaming per se. 

The section reads as if event streaming helps with the problem that services might not be available to respond. IMHO that is not the case.

Reply:
That's a fair point. Event-streaming is more of the "What" than the "how". I would say that Event-driven architecture is the how. Event streaming is just the streaming of events from point A to point B.